elements/xenon.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Sodium
+---
+
+Symbol : Xe  
+Atomic Number : 54
+
+1. Xenon is a purple glowing gas
+2. Xenon is one of the heavier noble gasses
+3. Xenon is not typically used in commertial lighting. 
```

Comment:
Format of page is correct. File name is correct. Good description added for element.